content/en/courses/350/poll-proj/_index.md

```diff
@@ -1,52 +1,55 @@
 ---
+
 # Page title
-title: The Poll Project Assignment
+العنوان: مهمة مشروع الاستطلاع
 
 # Title for the menu link if you wish to use a shorter link title, otherwise remove this option.
-linktitle: The Poll Project
+اسم الرابط: مشروع الاستطلاع
```

Comment:
@Mohammed-Alturki no need to change the files here above the --- mark as they are data not text. please modify your branch again and revert them back to english ... keep as arabic only the part to the right of the colon :

Reply:
Sure
  Thank you sir

Reply:
@Mohammed-Alturki make your changes directly to the branch `Mohammed-Alturki:Mohammed-Alturki-patch-1` then commit it ... it will be automatically included here for me to see ... you dont need to do anything else

Reply:
Great i will